packages/api/helper/mailer.js

```diff
@@ -18,12 +18,37 @@ const transport = {
 
 const transporter = nodemailer.createTransport(smtpTransport(transport));
 
+transporter.verify((error, success) => {
+  if (error) {
+    console.log(error);
+  } else {
+    console.log('All works fine, congratz!');
+  }
+});
+
 const email = new emailTemplate({
   transport: transporter,
   send: true,
+  //send status will eventually need to be updated to true
   preview: true,
 });
 
+function sendDeclineEmail(nomination) {
```

Comment:
Did you mean to combine the decline email work with this PR? It seems that the decline email is handled with PR https://github.com/the-difference-engine/ksf/pull/166